marginalia.el

```diff
@@ -70,6 +70,36 @@ It can also be set to an integer value of 1 or larger to force an offset."
   "Use whitespace margin for window widths larger than this value."
   :type 'integer)
 
+(defcustom marginalia-file-age-function #'marginalia--file-age
+  "Function which genrates a file age string from the file time."
+  :type 'function)
+
+(defcustom margialia-max-relative-age (* 60 60 24 14)
+  "Maximum relative age displayed, in seconds.
+
+Set to nil for there to be no maximum (always use a relative age),
+or 0 to never show a relative age."
+  :type '(choice (const nil) integer))
+
+(defcustom marginalia-absolute-age-function #'marginalia--absolute-age
+  "Function which genrates a file age string from the file time."
+  :type 'function)
+
+(defcustom marginalia-relative-age-function #'marginalia--relative-age
+  "Function which genrates a file age string from the file time."
+  :type 'function)
+
+(defcustom marginalia--relative-age-units
+  '(("s" . "sec")
+    ("m" . "min")
+    ("h" . "hour")
+    ("d" . "day")
+    ("y" . "year"))
+  "Expansions of the short units used by `seconds-to-string'.
+
+This is used in `marginalia--relative-age'."
+  :type '(alist :key-type string :value-type string))
```

Comment:
Why is this customizable and double-dashed?

Reply:
Let's get rid of that bogus dash.